plugins/discovery-file/build.gradle

```diff
@@ -53,6 +54,7 @@ integTestCluster {
     dependsOn setupSeedNodeAndUnicastHostsFile
     clusterName = 'discovery-file-test-cluster'
     extraConfigFile 'discovery-file/unicast_hosts.txt', srcUnicastHostsFile
+    module project(':modules:discovery-zen')
```

Comment:
This means discovery-file is going to load it's entire own copy of zen. I don't think that is what we want...